.github/workflows/build-and-push.yml

```diff
@@ -87,11 +86,13 @@ jobs:
         if: startsWith(github.ref, 'refs/tags/')
         run: |
 
-          echo TAG ="$(git describe --tags --abbrev=7)" >> "$GITHUB_ENV"
+          echo TAG="$(git describe --tags --abbrev=7)" >> "$GITHUB_ENV"
           # Version tags are deployed to prod.
           echo DEPLOYMENT_ENV=prod >> "$GITHUB_ENV"
           echo DEPLOYMENT_REALM=prod >> "$GITHUB_ENV"
 
+      - uses: docker/setup-buildx-action@v3
```

Comment:
not really .. the workflow didn't exist in the dev branch .. i pushed an older one... and then copied the one in the main ..since it can be used for all environments.